sdks/go/test/integration/wordcount/wordcount_test.go

```diff
@@ -30,16 +31,26 @@ import (
 
 func TestWordCount(t *testing.T) {
 	tests := []struct {
-		lines []string
-		words int
-		hash  string
+		lines                                                             []string
+		words                                                             int
+		hash                                                              string
+		smallWords                                                        string
+		lineLen                                                           string
```

Comment:
Remove the constants here and use them down in the test. We don't need per-test case repetitions of the same thing. The goal is for things to be as configurable as necessary, but not further, so we can *reduce* repetition.